packages/melos/lib/src/commands/analyze.dart

```diff
@@ -126,6 +139,10 @@ mixin _AnalyzeMixin on _Melos {
       options.add(fatalWarnings ? '--fatal-warnings' : '--no-fatal-warnings');
```

Comment:
Not releated to this PR, but if `fatalInfos` is applied that implies `--fatal-warnings` too right? So this should probably be in an `if else`

Reply:
Not really, the only way to have both is to intentionally pass them as arguments